script.js

```diff
@@ -19,25 +19,38 @@ setByteLengthPerOrigin = (origin, byteLength) => {
   localStorage.setItem('stats', JSON.stringify(statsJson));
 }
 
-headersReceivedListener = (requestDetails) => {
-  let filter = browser.webRequest.filterResponseData(requestDetails.requestId);
+isChromeExtension = () => {
+  return (typeof(browser) === 'undefined');
+}
 
-  filter.ondata = event => {
-    const origin = extractHostname(!requestDetails.originUrl ? requestDetails.url : requestDetails.originUrl);
-    setByteLengthPerOrigin(origin, event.data.byteLength);
+headersReceivedListener = (requestDetails) => {
+  if(isChromeExtension()){
```

Comment:
Code styling
```suggestion
  if (isChromeExtension()) {
```

Reply:
@supertanuki I can do this modification, but about formatting I have 2 options :

- make this specific modification.
- format the code with a code formatter like **Prettier** (included in my actual IDE VSCode), which lead to this specific modification + formatting the entire script.

What do you prefer?